src/main/java/org/jabref/gui/entryeditor/SourceTab.java

```diff
@@ -57,6 +62,32 @@
     private final ImportFormatPreferences importFormatPreferences;
     private final FileUpdateMonitor fileMonitor;
     private final DialogService dialogService;
+    private final CodeArea codeArea = new CodeArea();
```

Comment:
I fear this could interfer with tab switching and entry switching if you outline it as final variable

Reply:
Yes, you were right, I did not see that, but I fixed it now. Thanks.